packages/strapi-design-system/src/SubNav/SubNav.stories.mdx

```diff
@@ -0,0 +1,156 @@
+<!--- SubNav.stories.mdx --->
+
+import { useState } from 'react';
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { Applications, AddIcon, ApiTokens, AlertWarningIcon, AlertInfoIcon } from '@strapi/icons';
+import { matchSorter } from 'match-sorter';
+import { SubNav } from './SubNav';
+import { SubNavHeader } from './SubNavHeader';
+import { SubNavSection } from './SubNavSection';
+import { SubNavSections } from './SubNavSections';
+import { SubNavLink } from './SubNavLink';
+import { SubNavLinkSection } from './SubNavLinkSection';
+import { Box } from '../Box';
+import { Link } from '../Link';
+import { Row } from '../Row';
+import { TextButton } from '../TextButton';
+
+<Meta title="Design System/Organisms/SubNav" component={SubNav} />
+
+# SubNav
+
+This is the doc of the `SubNav` component
+
+## SubNav
+
+Description...
+
+<Canvas>
+  <Story name="base">
+    {() => {
+      const [search, setSearch] = useState('');
+      const links = [
+        {
+          id: 1,
+          label: 'Addresses',
+          icon: <AlertWarningIcon />,
+          to: '/address',
+        },
+        {
+          id: 2,
+          label: 'Categories',
+          to: '/category',
+        },
+        {
+          id: 3,
+          label: 'Cities',
+          icon: <Applications />,
+          to: '/city',
+        },
+        {
+          id: 4,
+          label: 'Countries',
+          to: '/country',
+          active: true,
+        },
+      ];
+      const filteredLinks = matchSorter(links, search, { keys: ['label'] });
```

Comment:
Not sure to understand why we need a third party lib?

Reply:
That was just a test, I'll remove it 👍🏽

Reply:
Yes that was for the CRA exemple right?